packages/pluggableWidgets/gallery-web/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gallery-web",
   "widgetName": "Gallery",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
I am not sure, but maybe bump a minor as it is a breaking change in DOM?

Reply:
Me to, let's ask @KevinVlaanderen or @cjfhodges ?